registration-processor/mosip-stage-executor/src/main/java/io/mosip/registration/processor/stages/executor/PropertySourceCustomizableEnvironment.java

```diff
@@ -0,0 +1,33 @@
+package io.mosip.registration.processor.stages.executor;
+
+import org.springframework.core.env.MutablePropertySources;
+import org.springframework.core.env.StandardEnvironment;
+
+/**
+ * The Class PropertySourceCustomizableEnvironment.
+ */
+final class PropertySourceCustomizableEnvironment extends StandardEnvironment{

```

Comment:
Yes. Since we override the behavior of the customizePropertySources in StandardEnvironment for setting the cloud property sources, we need this class.